src/contracts/interfaces/IIndexRegistry.sol

```diff
@@ -8,6 +8,12 @@ import "./IRegistry.sol";
  * @author Layr Labs, Inc.
  */
 interface IIndexRegistry is IRegistry {
+    // EVENTS
+    // emitted when an operator's index in at quorum operator list is updated
```

Comment:
typo: `in at` => `in the` (?)
I think also perhaps change 'quorum' to '`quorumNumber`th' ?

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/481daabb676aabe76b233992878a3ee566cf89e1